builtin/game/chat.lua

```diff
@@ -990,7 +990,7 @@ core.register_chatcommand("clearobjects", {
 
 core.register_chatcommand("msg", {
 	params = "<name> <message>",
-	description = "Send a private message",
+	description = "Send a message directly to a player",
```

Comment:
This doesn't sound right. Maybe this?

```suggestion
	description = "Send a direct message to a player",
```

or maybe even just

```suggestion
	description = "Send a message to a player",
```

`to a player` implies that it's direct.